internal/pkg/runtime/engines/singularity/prepare.go

```diff
@@ -37,12 +38,25 @@ func (e *EngineOperations) prepareUserCaps() error {
 
 	e.EngineConfig.OciConfig.SetProcessNoNewPrivileges(true)
 
-	file, err := capabilities.Open(buildcfg.CAPABILITY_FILE, true)
+	oldmask := syscall.Umask(0)
```

Comment:
Umask is required only in CLI commands, it was set previously in `capabilities.Open` for file creation